common/persistence/sql/sqlMatchingPersistence.go

```diff
@@ -328,6 +357,56 @@ VALUES
 
 	completeTransferTaskSQLQuery = `DELETE FROM transfer_tasks WHERE shard_id = :shard_id AND task_id = :task_id`
 
+	replicationTaskInfoColumns = `domain_id,
+workflow_id,
+run_id,
+task_id,
+type,
+first_event_id,
+next_event_id,
+version,
+last_replication_info`
+
+	replicationTaskInfoColumnsTags = `:domain_id,
+:workflow_id,
+:run_id,
+:task_id,
+:type,
+:first_event_id,
+:next_event_id,
+:version,
+:last_replication_info`
+
+	replicationTasksColumns = `shard_id, ` + replicationTaskInfoColumns
```

Comment:
For timer and transfer, it is useful to have a query without "shard_id" because then I can select directly into a `___TaskInfo` array. For replication tasks I am not able to do this because there is a field that I need to gob